### ISSUE TITLE
Find difference between two spec docs

### ISSUE THREAD

**Issue by Souvikns** [asyncapi/cli#2]
#### Reason/Context

Would be nice to have a dedicated set of commands related to finding differences in two spec documents and generate UI, or visualizing the changes right in the command line for a quick refference. 

This would provide a set of tools that could be used in the development process of event-driven process. 

> My motivation for raising this issue is to have a discussion on the features that the CLI would be implementing so as to have a clear picture of the AsyncApiDiff's use case

**Comment by derberg** [comment]
once diff library is ready, we definitely want to have it integrated here in the CLI. I don't think anyone thought on the vision on how to do that as diff is not started yet. Ink should give us a lot of flexibility in the form we do that.

**Comment by derberg** [comment]
closing in favour of https://github.com/asyncapi/cli/issues/58
